### PR TITLE
[10.0][ADD] product_country_restriction

### PR DESCRIPTION
**Pull request by rousseldenis**
Manage country restrictions for products

### CODE REVIEW

**Comment by rousseldenis** [comment]
Supersedes https://github.com/OCA/sale-workflow/pull/672

**Comment by pedrobaeza** [comment]
Please avoid plurals in module name

**Comment by rousseldenis** [comment]
WIP

**Comment by rousseldenis** [comment]
Needs review

**Comment by rousseldenis** [comment]
@Cedric-Pigeon

File: product_country_restriction/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class ResPartner(models.Model):
+
+    _inherit = 'res.partner'
+
+    country_restriction_id = fields.Many2one(
+        comodel_name='product.country.restriction',
+        string='Country Restriction',
+        default=lambda self: self._get_default_country_restrction_id(),
```
Comment (Cedric-Pigeon) [inline product_country_restriction/models/res_partner.py]: should be mandatory

**Review by Cedric-Pigeon — CHANGES_REQUESTED** [review]

**Comment by Cedric-Pigeon** [comment]
@rousseldenis Flake8 seems to be wrong

File: product_country_restriction/models/res_partner.py (hunk 1 above)
Comment (Cedric-Pigeon) [reply product_country_restriction/models/res_partner.py]: in fact no but you should at least fix the typo as well as in the method definition ;-)

**Review by Cedric-Pigeon — CHANGES_REQUESTED** [review]

**Review by Cedric-Pigeon — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
:+1:

**Comment by rousseldenis** [comment]
@OCA/core-maintainers

**Pull request merged**